bugsnag_test.go

```diff
@@ -612,15 +612,14 @@ func assertValidSession(t *testing.T, event *simplejson.Json, unhandled bool) {
 }
 
 func verifyExistsInStackTrace(t *testing.T, exception *simplejson.Json, exp *stackFrame) {
-	isFile := func(frame *simplejson.Json) bool { return getString(frame, "file") == exp.File }
+	isFile := func(frame *simplejson.Json) bool { return strings.HasSuffix(getString(frame, "file"), exp.File) }
 	isMethod := func(frame *simplejson.Json) bool { return getString(frame, "method") == exp.Method }
 	isLineNumber := func(frame *simplejson.Json) bool { return getInt(frame, "lineNumber") == exp.LineNumber }
-	isInProject := func(frame *simplejson.Json) bool { return getBool(frame, "inProject") == exp.InProject }
```

Comment:
Doesn't this change mean we're not testing if a frame is in project anymore?

Reply:
Yes and no. These test assertions were dependent on the location of the package while being tested, which made them fail unexpectedly if not in `$GOPATH/src`. There are several test cases for `Configuration.isProjectPackage()`, which underpins the `InProject` behavior. I'm also adding more tests as a part of some internals changes to resolve issues with `CallersFrames`.

Reply:
👍 that makes sense